conans/test/integration/toolchains/meson/test_install.py

```diff
@@ -0,0 +1,109 @@
+import os
+import textwrap
+
+from conans.test.assets.sources import gen_function_cpp, gen_function_h
+
+
+from ._base import TestMesonBase
+
+
+class MesonInstall(TestMesonBase):
+    _conanfile_py = textwrap.dedent("""
+    import os
+    import shutil
+    from conans import ConanFile, tools
+    from conan.tools.meson import Meson, MesonToolchain
+
+
+    class App(ConanFile):
+        settings = "os", "arch", "compiler", "build_type"
+        options = {"shared": [True, False], "fPIC": [True, False]}
+        default_options = {"shared": False, "fPIC": True}
+        exports_sources = "meson.build", "hello.cpp", "hello.h"
+        _meson = None
+
+        def config_options(self):
+            if self.settings.os == "Windows":
+                del self.options.fPIC
+
+        def generate(self):
+            tc = MesonToolchain(self)
+            # https://mesonbuild.com/Release-notes-for-0-50-0.html#libdir-defaults-to-lib-when-cross-compiling
+            tc.definitions["libdir"] = "lib"
```

Comment:
right now we don't manage install directories in build helpers (e.g. cmake)
problem meson uses `lib/x86_64-linux-gnu` by default on Linux.
I don't know if it belongs here, or better move it to the toolchain?

Reply:
This will probably be part of the "layouts" feature it it keeps moving forward